src/index.html

```diff
@@ -3,9 +3,26 @@
 <head>
 	<meta charset="UTF-8"/>
 	<meta name="viewport" content="width=device-width, initial-scale=1.0"/>
-	<meta name="description" content="SMARTHACK este un hackathon de 24 de ore în care mai multe echipe de câte 3-4 studenţi lucrează la dezvoltarea unui produs software pe baza unei teme, care se finalizează cu jurizarea şi premierea celor mai bune echipe."/>
-	<meta name="og:title" property="og:title" content="SmartHack 2019 - Hackathon">
+	<!-- Primary Meta Tags -->
+	<title>SmartHack 2020 - Hackathon</title>
+	<meta name="title" content="SmartHack 2020 - Hackathon">
+	<meta name="description" content="SMARTHACK este un hackathon de 24 de ore în care mai multe echipe de câte 3-4 studenţi lucrează la dezvoltarea unui produs software pe baza unei teme, care se finalizează cu jurizarea şi premierea celor mai bune echipe.">
 
+	<!-- Open Graph / Facebook -->
+	<meta property="og:type" content="website">
+	<meta property="og:url" content="https://smarthack.asmi.ro/">
+	<meta property="og:title" content="SmartHack 2020 - Hackathon">
+	<meta property="og:description" content="SMARTHACK este un hackathon de 24 de ore în care mai multe echipe de câte 3-4 studenţi lucrează la dezvoltarea unui produs software pe baza unei teme, care se finalizează cu jurizarea şi premierea celor mai bune echipe.">
+	<meta property="og:image" content="https://smarthack.asmi.ro/img/metadata.png">
```

Comment:
Aici trebuie schimbat linkul la imagine:
```suggestion
	<meta property="og:image" content="https://smarthack.asmi.ro/img/metatag.png">
```